### PR TITLE
[ingress-nginx] Adding an example with L2LoadBalancer

### PR DESCRIPTION
**Pull request by Beastlex**
## Description
Adding an example of usage ingress-nginx with L2LoadBalancer.

## Why do we need it, and what problem does it solve?
Examples will help improve the customer experience with the platform.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: ingress-nginx
type: chore
summary: Add an example of usage ingress-nginx with _L2LoadBalancer_ inlet.
impact_level: default
```

### CODE REVIEW

File: modules/402-ingress-nginx/docs/EXAMPLES.md [hunk 1]
```diff
@@ -128,4 +128,52 @@ metallb:
...
+spec:
+  addressPool:
+  - 192.168.2.100-192.168.2.150
+  nodeSelector:
+    node-role.kubernetes.io/loadbalancer: ""
+```
+
+Deploy the _IngressNginxController_ resource:
+* The **network.deckhouse.io/l2-load-balancer-name** annotation specifies the name _L2LoadBalancer_ (in the example _L2LoadBalancer_ with the name _ingress_ was created in the previous step)
+* The **network.deckhouse.io/l2-load-balancer-external-ips-count** annotation specifies how many addresses will be allocated from the pool described in _L2LoadBalancer_
```
Comment (apolovov) [inline modules/402-ingress-nginx/docs/EXAMPLES.md:163]:
```suggestion
* The **network.deckhouse.io/l2-load-balancer-external-ips-count** annotation specifies how many addresses will be allocated from the pool described in _L2LoadBalancer_.
```

File: modules/402-ingress-nginx/docs/EXAMPLES_RU.md [hunk 2]
```diff
@@ -128,4 +128,52 @@ metallb:
       value: frontend
 ```
 
+## Пример для bare metal (балансировщик L2LoadBalancer)
+
+Модуль [l2-load-balancer](../381-l2-load-balancer/) на текущий момент доступен только в редакции Enterprise Edition Deckhouse.
```
Comment (apolovov) [inline modules/402-ingress-nginx/docs/EXAMPLES_RU.md:133]: alert here

File: modules/402-ingress-nginx/docs/EXAMPLES_RU.md [hunk 3]
```diff
@@ -128,4 +128,52 @@ metallb:
...
+  name: ingress
+spec:
+  addressPool:
+  - 192.168.2.100-192.168.2.150
+  nodeSelector:
+    node-role.kubernetes.io/loadbalancer: "" # селектор узлов-балансировщиков
+```
+
+Создайте ресурс _IngressNginxController_:
+* В аннотации **network.deckhouse.io/l2-load-balancer-name** указывается имя _L2LoadBalancer_ (в примере _L2LoadBalancer_ с именем _ingress_ был создан на предыдущем шаге)
```
Comment (apolovov) [inline modules/402-ingress-nginx/docs/EXAMPLES_RU.md:162]:
```suggestion
* В аннотации **network.deckhouse.io/l2-load-balancer-name** указывается имя _L2LoadBalancer_.
```

File: modules/402-ingress-nginx/docs/EXAMPLES_RU.md [hunk 4]
```diff
@@ -128,4 +128,52 @@ metallb:
...
+* Аннотация **network.deckhouse.io/l2-load-balancer-external-ips-count** указывает сколько адресов будет выделено из пула, описанного в _L2LoadBalancer_
```
Comment (apolovov) [inline modules/402-ingress-nginx/docs/EXAMPLES_RU.md:163]:
```suggestion
* Аннотация **network.deckhouse.io/l2-load-balancer-external-ips-count** указывает сколько адресов будет выделено из пула, описанного в _L2LoadBalancer_.
```

**Review by apolovov — CHANGES_REQUESTED** [review]

File: modules/402-ingress-nginx/docs/EXAMPLES.md [hunk 5]
```diff
@@ -128,4 +128,52 @@ metallb:
       value: frontend
 ```
 
+## An example for Bare metal (with L2LoadBalancer)
+
+{% alert level="warning" %} This feature is available in Enterprise Edition only. {% endalert %}
+
+Enable the module:
```
Comment (apolovov) [inline modules/402-ingress-nginx/docs/EXAMPLES.md:135]:
```suggestion
Enable the `l2-load-balancer` module:
```

File: modules/402-ingress-nginx/docs/EXAMPLES.md [hunk 6]
```diff
@@ -128,4 +128,52 @@ metallb:
...
+metadata:
+ name: main
+spec:
+  ingressClass: nginx
+  inlet: LoadBalancer
+  loadBalancer:
+    annotations:
+      network.deckhouse.io/l2-load-balancer-name: ingress
+      network.deckhouse.io/l2-load-balancer-external-ips-count: "3"
+```
```
Comment (apolovov) [inline modules/402-ingress-nginx/docs/EXAMPLES.md:177]: Need a note, how to figure out the public ips.

```
kubectl -n d8-ingress-nginx get svc main
...
```

**Review by apolovov — CHANGES_REQUESTED** [review]

File: modules/402-ingress-nginx/docs/EXAMPLES_RU.md [hunk 7]
```diff
@@ -128,4 +128,52 @@ metallb:
       value: frontend
 ```
 
+## Пример для bare metal (балансировщик L2LoadBalancer)
+
+{% alert level="warning" %} Доступно только в Enterprise Edition. {% endalert %}
+
+Включите модуль:
```
Comment (apolovov) [inline modules/402-ingress-nginx/docs/EXAMPLES_RU.md:135]:
```suggestion
Включите модуль `l2-load-balancer`:
```

**Review by apolovov — APPROVED** [review]

**Review by apolovov — APPROVED** [review]

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [9338](https://github.com/deckhouse/deckhouse/pull/9338) to the branch [release-1.63](https://github.com/deckhouse/deckhouse/tree/release-1.63) successful!

**Pull request merged**